### DISCUSSION TITLE
Unable to delete github repo

### DISCUSSION THREAD

**Discussion by vishalm2213** [community/community#22485] [Projects and Issues]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/0/60a06ab5c2894d79689798c3c1bc8ff315563c54.png" data-download-href="/uploads/short-url/dMNwM7RslQ3AaSzfijNm4yVE0lK.png?dl=1" title="Screenshot 2022-02-28 034205" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/99785638/181071398-686098ff-a54a-4daf-83ee-acb9c19ccc6d.png" alt="Screenshot 2022-02-28 034205" data-base62-sha1="dMNwM7RslQ3AaSzfijNm4yVE0lK" width="690" height="455" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/0/60a06ab5c2894d79689798c3c1bc8ff315563c54_2_690x455.png, https://aws1.discourse-cdn.com/github/original/3X/6/0/60a06ab5c2894d79689798c3c1bc8ff315563c54.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/0/60a06ab5c2894d79689798c3c1bc8ff315563c54.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/0/60a06ab5c2894d79689798c3c1bc8ff315563c54_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot 2022-02-28 034205</span><span class="informations">828×547 46.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
Unable to delete GitHub repo.<br>
I want to delete this repo, but I am unable to do so.<br>
Please help.

**Comment by airtower-luna** [comment, marked as answer]
<p>The text you have typed isn’t exactly the same one that you’re asked to type to confirm. Compare the numbers! :wink:</p>

**Comment by vishalm2213** [comment]
<p>Thanks a lot for explaining, I finally understood my mistake. Extremely sorry for bothering you. It was a very silly mistake.</p>
